doc/topics/tutorials/minionfs.rst

```diff
@@ -4,10 +4,27 @@
 MinionFS Backend Walkthrough
 ============================
 
+Propagating Files
+============================
+
 .. versionadded:: 2014.1.0
 
 Sometimes, you might need to propagate files that are generated on a minion.
-Salt already has a feature to send files from a minion to the master:
+Salt already has a feature to send files from a minion to the master.
+
+Enabling Feature
+====================
+
+To use the propagate feature one configuration change is required on the
```

Comment:
I think this and line 19 could be worded slightly better, noting that `one configuration change is required` isn't necessary. Instead just say what the change and that it lives in the master config.